ProjectLighthouse.Servers.GameServer/Controllers/Slots/PublishController.cs

```diff
@@ -47,12 +46,16 @@ public async Task<IActionResult> StartPublish()
         if (slot == null)
         {
             Logger.Warn("Rejecting level upload, slot is null", LogArea.Publish);
+            await this.database.SendNotification(user.UserId,
+                "Level upload was rejected by the server because of an issue with your level.");
```

Comment:
```suggestion
            await this.database.SendNotification(user.UserId,
                "Your level failed to publish. (Error xx)");
```
I think all of these could benefit from better phrasing and specific error codes for each case. Instead of 'level upload' we should say 'level publish' and I don't think we should be saying was rejected by the server cause it sounds kind of weird. I'm willing to be flexible with this though.